canonical/ubuntu-pro-client
# Add support to esm-infra-legacy on Trusty

## Why is this needed?
This PR adds support to the new esm-infra-legacy service on Trusty.

## Test Steps
1. Launch a Trusty container
2. Set the contract url to staging in /etc/ubuntu-advantage/uaclient.conf
3. Attach by using the special token we have for testing
4. Certify that you can see `esm-infra-legacy`  on status, but the service is not enabled
5. Enable the service
6. Check apt-cache policy for the service url
7. Check that status reports the service as enabled
8. Disable the service
9. Check that esm-infra-legacy appears as disabled in status, but not esm-infra
10. Enable the service again
11. Run detach
12. Check that the machine is unattached now

## Checklist


## Does this PR require extra reviews?
